### ISSUE TITLE
Don't use babel-preset-es2015 (or maybe drop babel completely?)

### ISSUE THREAD

**Issue by typeofweb** [dsebastien/modernWebDevGenerator#104]
## Use `babel-preset-es2015-node4` instead of `babel-preset-es2015`

Since this generator's `package.json` specifies required nodejs version as `4.2.6`, there's no need to use `babel-preset-es2015`. ES2015 babel preset makes babel transpile all the codebase from ES2015 into ES5 even though node.js 4 can understand parts of ES2015 syntax. Therefore `babel-preset-es2015-node4` could be used instead to make the development process even faster.
## Alternative: Require nodejs 6 and drop babel completely

Nodejs v6 supports over 92% of ES2015 (according to [node.green](http://node.green)). If you agree to require nodejs v6 then there's no need to have babel as a dependency anymore. Nodejs v6 becomes active Long Time Support version in October 2016.

**Comment by dsebastien** [comment]
Well the main target of the generator are Web browsers. I'm not sure we can already forget about ES5 (?)

**Comment by typeofweb** [comment]
Yes, but as far as I understand the codebase, Babel is not used for client-side code. It's only used to transpile gulpfile, isn't it?

**Comment by dsebastien** [comment]
I get your point, you're right as far as the build of the generator itself is concerned; the same is true for the build of the modern-web-dev-build project :)
